### DISCUSSION TITLE
通过catalog查询hive表会出现卡死问题

### DISCUSSION THREAD

**Discussion by CanGuan** [apache/doris#22552] [A - General / Q&A]
我这边使用catalog查询hive表，出现查询某些表会卡死的问题(部分表成功)。在be fe日志都没发现异常，但是一直卡死。
此外，我中断查询之后，mysql client也没返回，fe有  kill query timeout 相关的warn日志。

请问各位大佬这种情况应该怎么定位&解决呢

**Comment by CanGuan** [comment]
貌似这个和struct里面的复杂类型有关。当查询含有struct<map<String, String>>的时候，会出现这个问题

**Comment by zy-kkk** [comment]
你的Doris是什么版本？

**Reply by CanGuan** [reply]
2.0-beta
